src/test/java/com/wikia/webdriver/elements/mercury/pages/InfoboxBuilderPage.java

```diff
@@ -136,9 +299,36 @@ public int countHeaders() {
     return headers.size();
   }
 
-  public String getBackgroundColor() {
-    wait.forElementVisible(component.get(0));
-    return component.get(0).getCssValue("background-color");
+  public InfoboxBuilderPage changeHeaderCollapsibilityState(int index) {
+    JavascriptExecutor js = (JavascriptExecutor) driver;
+    headers.get(index).click();
+
+    wait.forElementClickable(sidebarCheckbox);
+    sidebarCheckbox.click();
+
+    String script = "return window.getComputedStyle(document"
```

Comment:
FYI: soon we are going to merge to master the new method which could be useful here
https://github.com/Wikia/selenium-tests/pull/1120/files#diff-ceedb0f5093fdab9b86010a3e98a42b9R723